securedrop/source_app/utils.py

```diff
@@ -96,3 +99,7 @@ def normalize_timestamps(filesystem_id):
                 "Couldn't normalize submission "
                 "timestamps (touch exited with %d)" %
                 rc)
+
+
```

Comment:
We don't need to add random bytes. We only need to add size since after THS end-to-end encryption the bytes will look random anyway. Using `os.urandom` may exhaust our entropy, and we actually have checks that prevent key generation when there's insufficient entropy.

Reply:
Ok, I changed that to just using dots.